domenetjenester/behandling-klage/src/test/java/no/nav/foreldrepenger/behandling/kabal/KabalDokumenterTest.java

```diff
@@ -53,74 +54,116 @@ void setUp() {
     void testFinnerKlageOversendtFraBehandlingDokument() {
         var behandlingId = 1234L;
         var journalpost = new JournalpostId("12345");
-
         var behandlingDokument = opprettBehandlingDokumentEntitet(behandlingId, journalpost, DokumentMalType.KLAGE_OVERSENDT);
-
         when(behandlingDokumentRepository.hentHvisEksisterer(behandlingId)).thenReturn(Optional.of(behandlingDokument));
 
         var dokumentReferanses = kabalTjeneste.finnDokumentReferanserForKlage(behandlingId, SAKSNR,
             KlageResultatEntitet.builder().medKlageBehandlingId(behandlingId).build(), KlageHjemmel.ENGANGS);
 
-        sjekkResultat(dokumentReferanses, journalpost, TilKabalDto.DokumentReferanseType.OVERSENDELSESBREV);
+        assertThat(dokumentReferanses).hasSize(1);
+        assertThat(dokumentReferanses.getFirst().journalpostId()).isEqualTo(journalpost.getVerdi());
+        assertThat(dokumentReferanses.getFirst().type()).isEqualTo(TilKabalDto.DokumentReferanseType.OVERSENDELSESBREV);
+    }
+
+    @Test
+    void hivExceptionHvisOversendelsesbrevMangler() {
+        var behandlingId = 1234L;
```

Comment:
Scenariet som bør testes er at det finnes en dokumentBehandlingEntitet for oversendelsebrev-malen, men at journalpostId er null - bør gi exception